prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html

```diff
@@ -394,9 +394,14 @@
         {{ remoteAccessSite.site?.doingBusinessAs | default }}
 
       </ng-template>
-
+      <address>
+        {{ remoteAccessSite.site?.physicalAddress?.street | default }},
+        {{ remoteAccessSite.site?.physicalAddress?.city | default }}
+        {{ remoteAccessSite.site?.physicalAddress?.provinceCode | default }}
+        {{ remoteAccessSite.site?.physicalAddress?.postal | default }}
+      </address>
+      Vendor: {{ remoteAccessSite.site?.siteVendors[0]?.vendorCode | configCode: 'vendors' | default }}
```

Comment:
nvm i had an out of date branch, testing now whoops 👍

Reply:
cancel that still and error. You need an extra optional for that chaining to work.

```
remoteAccessSite.site?.siteVendors?.[0].vendorCode
```

Reply:
How it is written you don't verify siteVendors is not undefined before you try and index into it. You need one more conditional to make sure it's not the case.

Reply:
> How it is written you don't verify siteVendors is not undefined before you try and index into it. You need one more conditional to make sure it's not the case.

array optional operator is not supported in template. Used alternative approach to check.